### PR TITLE
fix: refactor article to post copy

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Refactor `article` copy to `post`

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1017 #done

### CODE REVIEW

File: packages/shared/src/components/post/PostContent.tsx [hunk 1]
```diff
@@ -150,7 +150,7 @@ export function PostContent({
           />
           <a
             href={post.permalink}
-            title="Go to article"
+            title="Go to post"
```
Comment (idoshamun) [inline packages/shared/src/components/post/PostContent.tsx:153]: This one sounds a bit funny worth double checking

File: packages/shared/src/components/post/PostModalActions.tsx [hunk 2]
```diff
@@ -94,7 +94,7 @@ export function PostModalActions({
       {onReadArticle && (
         <SimpleTooltip
           placement="bottom"
-          content="Read article"
+          content="Read post"
```
Comment (idoshamun) [inline packages/shared/src/components/post/PostModalActions.tsx:97]: This one as well

**Review by idoshamun — APPROVED** [review]

**Review by sshanzel — CHANGES_REQUESTED** [review]
Nice! Thanks for taking this on. I think there are still few left, just not sure if they should be kept.
- `ShareModal` the title is `Share article`.
- `sourceToMessage` in the `EnableNotification`.
- In the `PostToSquad` not sure what to do at line 71:  ```const title = `${post ? 'Share' : 'Post'} article`;```
- There is one in `AuthorStats` component.
- There is one in `PostsSection` component.
- The placeholder in `Comment.tsx`
- The `SelectArticle` component has many `article` words but not sure if they are intended.
- `/squads/utils.tsx` the `ModalState` is used as modal header.
- `SocialShare` last item has one.
- Some copies in the `featureManagement` file.

**Comment by rebelchris** [comment]
@sshanzel Thanks, somehow those didn't show up in my search, modified those and 2 more.

File: packages/shared/src/components/squads/SelectArticle.tsx [hunk 3]
```diff
@@ -49,12 +49,12 @@ export function SquadSelectArticle({
       <Modal.Body>
         <SquadTitle>
           {!hasData ? 'Read' : 'Share'}{' '}
-          <SquadTitleColor>an article</SquadTitleColor>
+          <SquadTitleColor>an post</SquadTitleColor>
```
Comment (sshanzel) [inline packages/shared/src/components/squads/SelectArticle.tsx:52]:
```suggestion
          <SquadTitleColor>a post</SquadTitleColor>
```

File: packages/shared/src/components/squads/utils.tsx [hunk 4]
```diff
@@ -11,8 +11,8 @@ import { UserShortProfile } from '../../lib/user';
 
 export enum ModalState {
   Details = 'Squad details',
-  SelectArticle = 'Pick an article',
-  WriteComment = 'Post article',
+  SelectArticle = 'Pick an post',
```
Comment (sshanzel) [inline packages/shared/src/components/squads/utils.tsx:14]:
```suggestion
  SelectArticle = 'Pick a post',
```

**Review by sshanzel — APPROVED** [review]
Just the suggestions then I think we should be good to go! Nice catch on the user page! 🚢

File: packages/shared/src/components/modals/SubmitArticleModal.spec.tsx [hunk 5]
```diff
@@ -79,7 +79,7 @@ it('should submit a valid URL', async () => {
   const input = (await screen.findByRole('textbox')) as HTMLInputElement;
   userEvent.type(input, link);
   input.value = link;
-  const btn = await screen.findByLabelText('Submit article');
+  const btn = await screen.findByLabelText('Submit post');
```
Comment (nimrodkra) [inline packages/shared/src/components/modals/SubmitArticleModal.spec.tsx:82]: Let's do it `Submit link` instead.

File: packages/shared/src/components/modals/SubmitArticleModal.spec.tsx [hunk 6]
```diff
@@ -99,8 +99,7 @@ it('should submit a valid URL', async () => {
     },
   });
 
-  const sent =
-    'You will be notified via email about the article request status';
+  const sent = 'You will be notified via email about the post request status';
```
Comment (nimrodkra) [inline packages/shared/src/components/modals/SubmitArticleModal.spec.tsx:102]: Change: We will notify you about the post-submission status via email

File: packages/shared/src/components/modals/SubmitArticleModal.spec.tsx [hunk 7]
```diff
@@ -221,7 +220,7 @@ it('should feedback already submitted article', async () => {
   const input = (await screen.findByRole('textbox')) as HTMLInputElement;
   userEvent.type(input, link);
   input.value = link;
-  const btn = await screen.findByLabelText('Submit article');
+  const btn = await screen.findByLabelText('Submit post');
```
Comment (nimrodkra) [inline packages/shared/src/components/modals/SubmitArticleModal.spec.tsx:223]: Submit link

File: packages/shared/src/components/modals/SubmitArticleModal.spec.tsx [hunk 8]
```diff
@@ -252,7 +251,7 @@ it('should feedback submitted article is deleted', async () => {
...
```
Comment (nimrodkra) [inline packages/shared/src/components/modals/SubmitArticleModal.spec.tsx:254]: Submit link

File: packages/shared/src/components/modals/SubmitArticleModal.tsx [hunk 9]
```diff
@@ -151,8 +151,8 @@ export default function SubmitArticleModal({
     return (
       <Alert className="mt-4" title="Submission guidelines">
         <AlertParagraph>
-          We want you to submit articles that are well thought out, high value
-          and high quality.
+          We want you to submit posts that are well thought out, high value and
```
Comment (nimrodkra) [inline packages/shared/src/components/modals/SubmitArticleModal.tsx:154]: We want you to submit links that are well thought out...

File: packages/shared/src/components/modals/SubmitArticleModal.tsx [hunk 10]
```diff
@@ -234,7 +234,7 @@ export default function SubmitArticleModal({
               <Alert
                 className="mt-4"
                 type={AlertType.Success}
-                title="You will be notified via email about the article request status"
+                title="You will be notified via email about the post request status"
```
Comment (nimrodkra) [inline packages/shared/src/components/modals/SubmitArticleModal.tsx:237]: We will notify you via email about the post submission status

File: packages/shared/src/lib/featureManagement.ts [hunk 11]
```diff
@@ -103,7 +103,7 @@ export class Features<T extends FeatureValue = string> {
 
   static readonly CompanionPermissionDescription = new Features(
     'companion_permission_description',
-    "We'll ask for extra permissions so we can show the companion directly on an article!",
+    "We'll ask for extra permissions so we can show the companion directly on an post!",
```
Comment (nimrodkra) [inline packages/shared/src/lib/featureManagement.ts:106]: We'll ask for extra permissions so we can show the companion directly on the original content!

File: packages/shared/src/components/squads/SelectArticle.tsx [hunk 12]
```diff
@@ -52,12 +52,12 @@ export function SquadSelectArticle({
...
           {!hasData ? 'Read' : 'Share'}{' '}
-          <SquadTitleColor>an article</SquadTitleColor>
+          <SquadTitleColor>a post</SquadTitleColor>
         </SquadTitle>
         <p className="py-4 text-center">
           {!hasData
-            ? 'Your reading history is empty! Please read at least one article by clicking on a post to start sharing with your Squad.'
-            : 'Pick an article that you would like to discuss with the members of your Squad.'}
+            ? 'Your reading history is empty! Please read at least one post by clicking on a post to start sharing with your Squad.'
+            : 'Pick an post that you would like to discuss with the members of your Squad.'}
```
Comment (nimrodkra) [inline packages/shared/src/components/squads/SelectArticle.tsx:60]: Pick n post that you would like to discuss with the members of your Squad.

**Pull request merged**